### ISSUE TITLE
proc.runq.defunct is two times larger than ps shows

### ISSUE THREAD

**Issue by test-account-0** [performancecopilot/pcp#110]
I can observe that on multiple machines. For example - I have two zombie processes:
```
root@x8:~# ps -efL H | grep Z
root       6829  200101    6829  0    1 Aug28 ?        Z      0:00 [git] <defunct>
root       97843  394347   97843  0    1  2015 ?        Z      0:00 [sh] <defunct>
```
and proc.runq.defunct=4.0

Why?

**Comment by goodwinos** [comment]
looks like that could be a bug - I'll investigate tomorrow if nobody beats me to it

**Comment by pcpemail** [comment]
We were calling refresh_global_pidlist() twice, resulting in all proc.runq
metrics being doubled. The attached patch fixes it, but may not be correct
for containers. Does anyone know if proc.runq.* metrics are supposed to be
valid for containers? Or are those metrics strictly global?

**Comment by pcpemail** [comment]
Have posted RH BZ 1372550 "proc.runq.* metric values are double counted"
(this is against RHEL, but the fix will be committed upstream first).

**Comment by natoscott** [comment]
commit 2fa04a366b532b2d165399dc025f57abf1725e71
Author: Nathan Scott <nathans@redhat.com>
Date:   Thu Sep 8 14:53:49 2016 +1000

    pmdaproc: fix double accounting of runq metrics
    
    Fix accidental double accounting of proc runq metrics, and
    add missing runq accounting call on the cgroups code path.
    
    Resolves Github Issue #110.
    Resolves Red Hat BZ #1372550.
